core/indigo-core/molecule/src/smiles_loader.cpp

```diff
@@ -3129,7 +3134,7 @@ void SmilesLoader::_readAtom(Array<char>& atom_str, bool first_in_brackets, _Ato
                 subatom = std::make_unique<QueryMolecule::Atom>(QueryMolecule::ATOM_NUMBER, element);
             else
             {
-                if (element_assigned)
+                if (element_assigned && !stereochemistry_options.ignore_errors)
```

Comment:
please add python test for this change

Reply:
As per company policy, bugs unrelated to the issue should not be included in the PR. The commit is going to be reverted